components/support-security/src/main/java/depromeet/ohgzoo/iam/security/WebConfig.java

```diff
@@ -40,7 +40,7 @@ public void addArgumentResolvers(List<HandlerMethodArgumentResolver> resolvers)
     @Override
     public void addCorsMappings(CorsRegistry registry) {
         registry.addMapping("/**")
-                .allowedOrigins("http://localhost:3000")
+                .allowedOrigins("http://localhost:3000", "https://11th-5team-fe.vercel.app")
```

Comment:
```suggestion
                .allowedOrigins("http://localhost:3000", "https://11th-5team-fe.vercel.app","http://192.168.35.245:3000")
```

모바일웹에서 테스트해야돼서 제 ip도 추가해주시면 감사하겠습니다~